### ISSUE TITLE
Content search query builder improvements 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#18316]
In the content search portlet if you try to search by a piece of content and then try to get the query, we are having a long query here that include some clause that maybe are not necessary. 

Here the query:
```
+contentType:webPageContent +(conhost:48190c8c-42c4-46af-8d1a-0cd5db894797 conhost:SYSTEM_HOST) +catchall:engl* title:'engl'^15 title:engl^5 title_dotraw:*engl*^5 +languageId:1 +deleted:false +working:true
```
### Expected behavior
We have 2 title clauses and also we should not include the ^15 and ^5 parameters here. 

### Screenshots
![image](https://user-images.githubusercontent.com/2641437/79490437-18916000-7fda-11ea-9870-177d319a8cc1.png)

### Desktop (please complete the following information):
Tested on master // postgress // FF

**Comment by nollymar** [comment]
PR: https://github.com/dotCMS/core/pull/18471

**Note to QA: ** Please re-test this issues as well: 
https://github.com/dotCMS/core/issues/17401
https://github.com/dotCMS/core/issues/17163
https://github.com/dotCMS/core/issues/16161

**Comment by fabrizzio-dotCMS** [comment]
I tested #17401 #17163 and #16161 
They all are working as they should.

Lucene Queries are generated fine. I didn't get to see any repetitive types or fields listed.
All filters can be applied successfully. Even for different sites.

My Only concern here is that now we're listing the Host as another Content type 
And since we're doing that we should probably include that type within the set of Types displayed on the right-hand sidebar drop-down. 

![Screen Shot 2020-06-01 at 14 17 18](https://user-images.githubusercontent.com/37148755/83462697-1d469200-a429-11ea-9727-cab20d7cb712.png)
.
 
Also Not sure what must happen to the actions associated with the Host.
if we select the content of type Host what actions should be listed there? 
Can we archive hosts from the bulk action dialogs?

**Comment by nollymar** [comment]
We agreed to add the `Host` filter back to avoid the issue reported by @fabrizzio-dotCMS .
This is the PR: https://github.com/dotCMS/core/pull/18579

**Comment by fabrizzio-dotCMS** [comment]
Host is no longer present in the results.

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.1 .// Postgres // FF
